packages/orbit-components/src/InputField/__tests__/index.test.tsx

```diff
@@ -142,14 +142,14 @@ describe("InputField", () => {
       expect(input).toHaveAttribute("data-state", "error");
       expect(input).toBeInvalid();
 
-      await user.tab();
+      await act(() => user.tab());
```

Comment:
I think it should be the opposite, we removed recently act wrappers 
```suggestion
      await user.tab();
```